lib/jekyll-sitemap.rb

```diff
@@ -23,9 +29,9 @@ def generate(site)
       end
     end
 
-    # Array of all non-jekyll site files with an HTML extension
-    def html_files
-      @site.static_files.select { |file| File.extname(file.relative_path) == ".html" }
+    # Array of all non-jekyll site files with given extensions
+    def exts_files (exts=[])
```

Comment:
Can you call this `filtered_static_files` or `sitemap_static_files` to make it more direct?